frontend/src/components/EntityListTable.vue

```diff
@@ -18,7 +18,7 @@
       </tr>
       </thead>
       <tbody>
-      <tr v-for="e in entities" :key="e.id">
+      <tr v-for="e in entities" :key="e.id" :class="{'table-light': e.deleted}">
```

Comment:
Forgive me, as I'm new to Vue.js, but I don't get this change... we're binding the class attribute (so far, so good), but not to a state or string, but to an to an object, which has `'table-light'` as its only property and the value of it is a boolean, indicating whether the entity `e` is marked as deleted or not... what does this do?

Does it somehow set `class="table-light"` if `e` is deleted?

Reply:
Yes, exactly. That is a [special case](https://vuejs.org/guide/essentials/class-and-style.html) of bindings.

Reply:
OK, good to know.